src/doublylinkedlist.c

```diff
@@ -110,8 +113,47 @@ void DList_InsertHeadList(PDLIST_ENTRY listHead, PDLIST_ENTRY entry)
 
 IMPLEMENT_MOCKABLE_FUNCTION(, int, DList_ForEach, PDLIST_ENTRY, listHead, DLIST_ACTION_FUNCTION, actionFunction, void*, actionContext)
 {
-    (void)listHead;
-    (void)actionFunction;
-    (void)actionContext;
-    return 0;
+    int result = 0;
```

Comment:
>= 0; [](http://example.com/codeflow?start=15&length=4)

function result should not be initialized here.

Reply:
Removed.